upgrade/upgrading-monorepo.md

```diff
@@ -206,7 +204,9 @@ Typical upgrade sequence should be:
     - `build.dockerfile` should be `docker/Dockerfile`
     - execute `docker-compose up -d --build`, microservices should be up and running
 
-[From v10.0.0 to v11.0.0-dev]: https://github.com/shopsys/shopsys/compare/v10.0.0...master
+[From v10.0.2 to 10.0.3-dev]: https://github.com/shopsys/shopsys/compare/v10.0.2...rc-10-0-2
+[From v10.0.2 to v10.0.2]: https://github.com/shopsys/shopsys/compare/v10.0.2...v10.0.2
```

Comment:
it feels like this line should not be here

Reply:
I have removed all these links as they're not used anywhere :)